osbrain/tests/test_agent_sync_publications.py

```diff
@@ -59,6 +60,13 @@ def reply(self, delay):
         return 'x' + str(delay)
 
 
+class ServerNumbers(Agent):
+    def publish(self):
+        self.each(0.1, 'send', 'pub', 1, topic='negate')
+        self.each(0.1, 'send', 'pub', 2, topic='normal')
+        self.each(0.1, 'send', 'pub', 'not received', topic='topic_C')
+
```

Comment:
If you want to make sure messages are sent in a given order, why don't you do:

```python
def publish(self):
    self.send('pub', 1, topic='negate')
    self.send('pub', 2, topic='normal')
    self.send('pub', 'not received', topic='topic_C')

# [...]
each(0.1, 'publish')
```

?

Just started from here my review, so maybe I'll find later something that makes me understand this better... :sweat_smile: